.github/workflows/build-and-test-x86-extra.yml

```diff
@@ -0,0 +1,77 @@
+name: build and run extra tests on x86-64
+on:
+  # * is a special character in YAML so we quote lines containing it
+  push:
+    branches:
+      - '*argon*' # Run when pushing branches that have argon in the name
+  schedule:
+  - cron: '0 12 * * 1-5' # Run at 12pm UTC (5am Pacific Time) Monday-Friday
+jobs:
+  test-on-ubuntu-latest:
+    strategy:
+      matrix:
+        target: [
+          "x86_64-unknown-linux-gnu"
+        ]
+        build: [ # we do release with overflow checks since debug takes forever
+          # release build with overflow checks without optimized assembly routines
+          {name: "release", flags: "--release -Coverflow-checks --no-default-features --features=bitdepth_8,bitdepth_16"},
+          # release build with overflow checks and optimized assembly routines
+          {name: "release", flags: "--release -Coverflow-checks"}
+        ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: install prerequisites
+        uses: awalsh128/cache-apt-pkgs-action@latest
+        with:
+          packages: meson nasm gcc-multilib
+          version: 1.0 # version of cache to load
+      - name: git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: cache rust toolchain
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.rustup/toolchains
+            ~/.rustup/update-hashes
+            ~/.rustup/settings.toml
+          key: ${{ runner.os }}-${{ matrix.target }}-rust-toolchain-${{ hashFiles('rust-toolchain', 'rust-toolchain.toml') }}
+      - name: cache rust crates
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - name: cache argon test vectors
+        id: cache-argon
+        uses: actions/cache@v3
+        with:
+          path: |
+            tests/argon
+          key: argon_coveragetool_av1_base_and_extended_profiles_v2.1.1.zip
+      - name: cargo build for ${{ matrix.target }} ${{ matrix.build.name }}
+        run: |
+          cargo clean
+          rustup target add ${{ matrix.target }} --toolchain nightly-2022-08-08
```

Comment:
I don't think it's worth fixing right now in this PR since it's already done elsewhere, but I don't think we need to hardcode the toolchain version here.  It's specified in our `rust-toolchain.toml` and we'd like to keep that as the single source of truth so nothing gets out of sync.

Reply:
Yeah, I agree. I removed them and it looks like we still get the desired toolchain (m worry was that we'd pull down the stable toolchain for no reason) https://github.com/memorysafety/rav1d/pull/236